internal/p2p/peermanager.go

```diff
@@ -271,7 +272,7 @@ type PeerManager struct {
 }
 
 // NewPeerManager creates a new peer manager.
-func NewPeerManager(selfID types.NodeID, peerDB dbm.DB, options PeerManagerOptions) (*PeerManager, error) {
+func NewPeerManager(selfID types.NodeID, externalAddr string, peerDB dbm.DB, options PeerManagerOptions) (*PeerManager, error) {
```

Comment:
Seems like we should add this option to the `PeerManagerOptions` struct rather than passing in another argument.

Reply:
perfect, will do. i actually considered this originally and took a look at PeerManagerOptions but for whatever reason had the feeling it didn't quite fit, possibly just because i was too tired :laughing:

Reply:
solved in e38f0be5ba227347f89b203d25a77e959cc3e0be